eth-contracts/contracts/service/ClaimFactory.sol

```diff
@@ -116,6 +119,16 @@ contract ClaimFactory is RegistryContract {
             _claimer,
             fundBlock);
 
+        (uint spMin, uint spMax) = ServiceProviderFactory(
+            registry.getContract(serviceProviderFactoryKey)
+        ).getAccountStakeBounds(_claimer);
```

Comment:
This will succeed but the tx will revert in claimRewards which is the ONLY allowed caller for this - not sure it matters tbh. reasonable fix might be to just check that beforehand but same behavioral result